src/renderer/components/dex/MarketSelection.vue

```diff
@@ -60,6 +64,12 @@ export default {
     storeStateCurrentMarket() {
       return this.$store.state.currentMarket;
     },
+    baseCurrencyUnitPrice() {
+      return neo.getHolding(this.storeStateCurrentMarket.baseAssetId).unitValue;
```

Comment:
All services are available via a mixin to each component.

```javascript
return this.$services.neo.getHolding(this.storeStateCurrentMarket.baseAssetId).unitValue;
```

Reply:
Fixed. Thanks for the help.